src/redux/actions/concepts/index.js

```diff
@@ -5,7 +5,7 @@ import { isErrored, fetchConcepts, isFetching } from './ConceptActionCreators';
 const fetchConceptsAction = (
   query = '',
   names = [],
-  limit = 25,
+  limit = 500,
```

Comment:
May I know why the limit was changed from 25 to 500? Isn't it much?

Reply:
@emmabaye I wanted to get more concepts so as to test the limit per page because I realized with a limit of 25 I could not tell if it affects the application functionality and also the time taken to fetch the concepts.

Reply:
A suggestion for this would be to provide a user with the ability to change the page limit.